php_zookeeper.c

```diff
@@ -856,8 +856,6 @@ static void php_zk_destroy(php_zk_t *i_obj TSRMLS_DC)
 {
 	php_zk_close(i_obj TSRMLS_CC);
 	zend_hash_destroy(&i_obj->callbacks);
-
-	efree(i_obj);
```

Comment:
I think you're right about this.
See: https://bugs.php.net/bug.php?id=77692

Reply:
I've fixed it in 0.7.1.
Thank you for your effort anyway.

Reply:
with previous versions I was unable to update the Symfony builds but with this version it works so I guess you caught it https://github.com/symfony/symfony/pull/30486 Thanks!